zerver/tests/test_events.py

```diff
@@ -2440,12 +2440,15 @@ def do_set_realm_property_test(self, name: str) -> None:
             state_change_expected = True
             old_value = vals[count]
             num_events = 1
+            print(name)
             if name == "email_address_visibility" and Realm.EMAIL_ADDRESS_VISIBILITY_EVERYONE in [
                 old_value,
                 val,
             ]:
                 # email update event is sent for each user.
                 num_events = 11
+            if name == "waiting_period_threshold":
+                num_events = 2
```

Comment:
@refeed I don't know, that's why i asked about that in CZO, i had to change it to that, because otherwise the test doesn't pass, but i don't know why this is the case..

Reply:
I see, sometimes the tests in gh actions are flaky too and needs to be restarted. I just tested in my local env it is actually this change that made the test fails. I think it is safe to revert all the changes in this file

Reply:
@refeed okay, that really seemed to be the problem..
But the test failed a few days ago locally too, do you have an idea why this could be the case? (now it is working fine, maybe because i rebased or provisioned?)